### ISSUE TITLE
Atom feature dimension in qm9_example.ipynb

### ISSUE THREAD

**Issue by nd-02110114** [materialsvirtuallab/megnet#18]
I wonder why the atom feature dimension becomes `21` in qm9_exmaple.ipynb
According to your thesis, I seem that the atom feature dimension becomes `19`

Atom type: 5 
Chirality: 2
Ring sizes: 6
Hybridization: 3
Acceptor: 1
Donor: 1
Aromatic: 1

Total: 5+2+6+3+1+1+1 = 19 dim

Is this example correct..?
https://github.com/materialsvirtuallab/megnet/blob/master/notebooks/qm9_example.ipynb

**Comment by chc273** [comment]
@nd-02110114 

Try this:

```
fl = ["type", "chirality", "ring_sizes", "hybridization", "donor", "acceptor", "aromatic"]
for i, j in enumerate(fl):
    new_data = []
    for k in atom_feature:
        new_data.append([[m[i]] for m in k])
    atom_fc = FeatureClean(categorical=["type", "chirality", "hybridization", "donor", "acceptor", "aromatic"],
                       feature_labels=[j])
    print(j, atom_fc.fit_transform(new_data)[0].shape)
```

You can see the dimension is like the following 

```
type (5, 4)
chirality (5, 1)
ring_sizes (5, 9)
hybridization (5, 4)
donor (5, 1)
acceptor (5, 1)
aromatic (5, 1)
```

Anyways, `qm9.py` is only meant as an example, will be deprecated eventually. Check out molecule examples.
